bosco_cluster

```diff
@@ -36,19 +36,18 @@ TR=tr
 usage()
 {
 cat << EOM
-usage: $0 [options] command
+usage: $0 command
 
 commands:
  -l|--list                  List the installed clusters
  -a|--add host sched        Install and add a cluster, with scheduler sched
  -r|--remove [host]         Remove the installed cluster (first in list)
  -s|--status [host]         Get status of installed cluster
  -t|--test [host]           Test the installed cluster (all clusters)
- -h|--help                  Show this help message
-
-options
```

Comment:
are you not a fan of the logical separation of commands and options?

i think the point is this script takes exactly one command and zero or more options.

Reply:
Ah, I actually missed that change (that commit just brings our version up to date with upstream's `bosco_cluster`). I like the separation but I guess I prefer our version to match upstream wherever possible

Reply:
Ok i agree with your philosophy in that case.  Though now i feel like making an upstream PR to undo whatever introduced that, um, damage :-x